exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileResources.java

```diff
@@ -60,8 +63,7 @@
 @RolesAllowed(DrillUserPrincipal.AUTHENTICATED_ROLE)
 public class ProfileResources {
   static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(ProfileResources.class);
-
-  public final static int MAX_PROFILES = 100;
+  private static Integer MaxProfiles = null;
```

Comment:
This can be an int. The config system requires that you set some value for the config property, unless you go through some hokey-pokey to check if the property exists.